tests/integration/sourcemaps/upload.rs

```diff
@@ -1,6 +1,66 @@
 use crate::integration::{mock_endpoint, register_test, EndpointOptions};
 use mockito::{server_url, Mock};
 
+enum ServerBehavior {
+    Legacy,
+    Modern,
+}
+
+// Endpoints need to be bound, as they need to live long enough for test to finish
+fn mock_common_upload_endpoints(behavior: ServerBehavior) -> Vec<Mock> {
+    let (accept, release_request_count, assemble_endpoint) = match behavior {
+        ServerBehavior::Legacy => (
+            "\"release_files\"",
+            2,
+            "/api/0/organizations/wat-org/releases/wat-release/assemble/",
+        ),
+        ServerBehavior::Modern => (
+            "\"release_files\", \"artifact_bundles\"",
+            0,
+            "/api/0/organizations/wat-org/artifactbundle/assemble/",
+        ),
+    };
+    let chunk_upload_response = format!(
+        "{{
+            \"url\": \"{}/api/0/organizations/wat-org/chunk-upload/\",
+            \"chunkSize\": 8388608,
+            \"chunksPerRequest\": 64,
+            \"maxRequestSize\": 33554432,
+            \"concurrency\": 8,
+            \"hashAlgorithm\": \"sha1\",
+            \"accept\": [{}]
+          }}",
+        server_url(),
+        accept,
+    );
+
+    vec![
+        // bad endpoint, bad endpoint
```

Comment:
Not sure what those comments mean here

Reply:
That's from an earlier version, removed.